contents/docs/llm-analytics/installation/langchain.mdx

```diff
@@ -109,4 +151,10 @@ phClient.shutdown();
 
 </MultiLanguage>
 
-This automatically captures many properties into PostHog including `$ai_input`, `$ai_input_tokens`, `$ai_latency`, `$ai_model`, `$ai_model_parameters`, `$ai_output_choices`, and `$ai_output_tokens`. It also automatically creates a trace hierarchy based on how LangChain components are nested 
\ No newline at end of file
+This automatically captures many properties into PostHog including `$ai_input`, `$ai_input_tokens`, `$ai_latency`, `$ai_model`, `$ai_model_parameters`, `$ai_output_choices`, and `$ai_output_tokens`. It also automatically creates a trace hierarchy based on how LangChain components are nested.
```

Comment:
I think I mentioned above, maybe this should be a table with some additional explanations for what these are. + a note that these can be used during insight/dataware house analysis